Dockerfile

```diff
@@ -12,6 +12,7 @@ RUN set -ex; \
         curl \
         lxc \
         iptables \
+        liblzma-dev \
```

Comment:
How widely installed is liblzma? We should make sure it exists on all the main already-supported Linux distros and OS X.